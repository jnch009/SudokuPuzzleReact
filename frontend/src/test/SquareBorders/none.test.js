import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';
import NoBorder from '../../Components/Square/squareBorders/None';

it('No border', () => {
  const { container } = render(<NoBorder modify={true} />);
  expect(container.firstChild).toMatchSnapshot();
});

it('No border not modifiable', () => {
  const { container } = render(<NoBorder modify={false} />);
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).toHaveStyleRule('color', '#fff');
  expect(container.firstChild).toHaveStyleRule('cursor', 'not-allowed!important');
});
