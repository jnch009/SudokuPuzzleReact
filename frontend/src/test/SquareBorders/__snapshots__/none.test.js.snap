// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No border 1`] = `
<button
  class=""
/>
`;

exports[`No border not modifiable 1`] = `
.c0 {
  color: #fff;
  cursor: not-allowed!important;
}

<button
  class="c0"
/>
`;
